_topic_maps/_topic_map.yml

```diff
@@ -73,6 +73,8 @@ Topics:
   File: ossm-multi-cluster-topologies
 - Name: Deploying multiple service meshes on a single cluster
   File: ossm-deploying-multiple-service-meshes-on-single-cluster
+- Name: Istioctl
```

Comment:
Can we consider a more descriptive title that's similar to the others, like "Installing Istio's command-line utility: Istioctl" or just "Installing Istio's command-line utility"?

Reply:
done